### PR TITLE
Update cloud native principles

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

**Review by wvwatson — APPROVED** [review]

**Review by taylor — APPROVED** [review]

**Review by denverwilliams — APPROVED** [review]

**Review by lixuna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 
 **How are** [**cloud native**](cloud-native-principles.md) **systems loosely coupled?**
 
-Cloud native systems have a clear separation between their processes \[2\]. These [microservices](cloud-native-microservice-principles.md) usually use a technology like containers and aim for one process per container \[3\]. Cloud native applications also have all of their dependencies packaged with them during the build phase which is then used for deployment \[4\].
+Cloud native systems have a clear separation between their processes \[2\]. They utilize the Unix philosphy of doing one thing and doing it well. These [microservices](cloud-native-microservice-principles.md) usually use a technology like containers and aim for one process per container \[3\].
```
Comment (jeffsaelens) [inline README.md:10]: I don't think we should remove mention of CN apps bringing their dependencies with them. This speaks directly to being platform/infrastructure agnostic. Maybe capture both thoughts?

"Cloud native systems have a clear separation between their processes \[2\]. They utilize the Unix philosphy of doing one thing and doing it well. These [microservices](cloud-native-microservice-principles.md) usually use a technology like containers and aim for one process per container \[3\]. As such, cloud native applications should have all of their dependencies packaged with them during the build phase and leveraged during deployment \[4\].

File: README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
 **How are cloud native systems deployed?**
 
-Cloud native applications are deployed onto immutable infrastructure \(generic host servers that support orchestration \[7\]\). Cloud native applications are not changed after deployment. New features for an application are rolled out as new artifacts and configuration \(e.g. containers\)
+Cloud native applications are deployed onto immutable infrastructure \(generic host servers that support orchestration \[7\]\). Cloud native applications also have all of their dependencies packaged with them during the build phase which is then used for deployment \[4\]. Cloud native applications are not changed after deployment. New features for an application are rolled out as new artifacts and configuration \(e.g. containers\)
```
Comment (jeffsaelens) [inline README.md]: I think dependencies should stay up top. The part about cloud native applications not being changed after deployments needs more nuance around the Day 0 and Day N/build vs app data configuration discussions.

**Review by jeffsaelens — CHANGES_REQUESTED** [review]
A few tiny tweaks then good for merge.

File: README.md (hunk 1 above)
Comment (wvwatson) [reply README.md:10]: @xmulligan Do you think you could resubmit with this changes?

File: README.md (hunk 2 above)
Comment (xmulligan) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (xmulligan) [reply README.md:10]: done

**Review by jeffsaelens — APPROVED** [review]
Updates look good.

**Pull request merged**